src/entity/active_model.rs

```diff
@@ -1423,8 +1423,8 @@ mod tests {
             vec![
                 Transaction::from_sql_and_values(
                     DbBackend::Postgres,
-                    r#"UPDATE "fruit" SET  WHERE "fruit"."id" = $1 RETURNING "id", "name", "cake_id""#,
```

Comment:
Didn't know we put such a test case here

Reply:
Yep. That's bad :(

Reply:
It's a red flag that no one notice